### ISSUE TITLE
Remove button shows when uploading image not completed successfully.

### ISSUE THREAD

**Issue by dbrezoev** [uploadcare/angular-uploadcare#17]
When uploading a local file, and I press the Done button, before the file is processed fully. (The circle in the upper left corner is not complete), my ng-model property did not succeed to bind its value, but Remove button appears. I do not have the url but it acts like I do. So if it is not a bug, how can I disable the Done button until image is ready? Thanks in advance.

**Comment by dbrezoev** [comment]
Yes I use remove when clearable is enabled. And sorry for the inconvenience you can delete  this. Thanks for the quick response.

**Comment by dmitry-mukhin** [comment]
@Zmoki @homm what happens when user clicks remove button before files are uploaded?
